models/DataObject/Concrete.php

```diff
@@ -663,7 +663,7 @@ public static function __callStatic($method, $arguments)
 
             $list = static::makeList($listConfig, $objectTypes);
 
-            if ($field instanceof AbstractRelations && $field->isFilterable()) {
```

Comment:
why removed the `isFilterable()` check?

Reply:
See https://github.com/pimcore/pimcore/blob/9c2c366e7b96b5994828bb8c33d5561737cb7383/models/DataObject/Concrete.php#L610-L612